bot.js

```diff
@@ -1000,6 +1000,8 @@ bot.on('message', function (user, userID, channelID, message, evt) {
 					uss = message.substring(12, 30)
 				} else  if (message.length == 32){
 					uss = message.substring(13, 31)
+				} else  if (message.length == 27){
+					uss = message.substring(10, 27)

```

Comment:
I'm gonna look at this, see what the heck you are trying to do, but likely this is useless

Reply:
Ok, so I read your description and see what you're trying to do. But still, no. People don't use IDs regularly. Not all of us are developers.